### PR TITLE
#2496: Onboarding tab

### PR DESCRIPTION
**Pull request by psymoon**
## Checklist

- [ ] Confirm the **acceptance criteria** is fully satisfied in the issue(s) this PR will close
- [ ] Add thorough **tests** or an explanation of why it does not
- [ ] Add a **CHANGELOG entry** if applicable
- [ ] Add **QA labels** on the associated issue (not this PR; `qa-ready` or `qa-notneeded`)

### CODE REVIEW

File: app/src/main/java/org/mozilla/tv/firefox/utils/Settings.kt [hunk 1]
```diff
@@ -55,6 +57,9 @@ class Settings private constructor(context: Context) {
     fun shouldShowTurboModeOnboarding(): Boolean =
             !preferences.getBoolean(OnboardingActivity.ONBOARD_SHOWN_PREF, false)
 
+    fun shouldShowFxaOnboarding(): Boolean =
+            !preferences.getBoolean(FXA_ONBOARD_SHOWN_PREF, false)
```
Comment (Baron-Severin) [inline app/src/main/java/org/mozilla/tv/firefox/utils/Settings.kt]: This should also check that the user is in the experiment.

File: app/src/main/java/org/mozilla/tv/firefox/navigationoverlay/NavigationOverlayFragment.kt [hunk 2]
```diff
@@ -276,6 +277,11 @@ class NavigationOverlayFragment : Fragment() {
                     AccountState.AuthenticatedNoProfile -> {
                         fxaButton.setImageResource(R.drawable.ic_avatar_authenticated_no_picture)
                         fxaButton.contentDescription = resources.getString(R.string.fxa_navigation_item_signed_in2)
+
+                        val settings = Settings.getInstance(context!!)
+                        if (settings.shouldShowFxaOnboarding()) {
+                            navigationOverlayViewModel.showFxaOnboardingScreen(context!!)
```
Comment (Baron-Severin) [inline app/src/main/java/org/mozilla/tv/firefox/navigationoverlay/NavigationOverlayFragment.kt]: It looks like this code should work to me, but I created a new FF account, logged in here, then verified my account, and I didn't get the onboarding popup.  I don't have any account where I can test [this expected behavior](https://github.com/mozilla-mobile/firefox-tv/issues/2496#issuecomment-528434160), were you able to verify that it works as expected?

Comment (psymoon) [reply app/src/main/java/org/mozilla/tv/firefox/navigationoverlay/NavigationOverlayFragment.kt]: This is working as expected for me, weird :/

**Comment by psymoon** [comment]
# Request for data collection review form

**All questions are mandatory. You must receive review from a data steward peer on your responses to these questions before shipping new data collection.**

1) What questions will you answer with this data?
* Probe to track how many users see the Fxa (receive tab) onboarding screen

2) Why does Mozilla need to answer these questions?  Are there benefits for users? Do we need this information to address product or business requirements? Some example responses:
* FFTV team wants to be able to tell how many people are seeing the Fxa onboarding screen

3) What alternative methods did you consider to answer these questions? Why were they not sufficient?
* This is the only way to find out this information.

4) Can current instrumentation answer these questions?
* No telemetry currently exists to answer these questions.

5) List all proposed measurements and indicate the category of data collection for each measurement, using the Firefox [data collection categories](https://wiki.mozilla.org/Firefox/Data_Collection) on the Mozilla wiki.   

**Note that the data steward reviewing your request will characterize your data collection based on the highest (and most sensitive) category.**

<table>
  <tr>
    <td>Measurement Description</td>
    <td>Data Collection Category</td>
    <td>Tracking Bug #</td>
  </tr>
  <tr>
    <td>Probe to track how many users see the Fxa onboarding screen</td>
    <td>2</td>
    <td>#2496</td>
  </tr>
</table>

6) How long will this data be collected?  Choose one of the following:
* @athomasmoz will permanently monitor this data.

7) What populations will you measure?
* All users with telemetry enabled, on release channel.

8) If this data collection is default on, what is the opt-out mechanism for users?
* There is an option in settings to stop sending usage data.

9) Please provide a general description of how you will analyze this data.
* The data will be analyzed through Redash queries and dashboards.

10) Where do you intend to share the results of your analysis?
* On Redash and with mobile teams.

**Pull request merged**